services/catalog/src/simcore_service_catalog/api/routes/services_access_rights.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+from fastapi import APIRouter, Depends, Header
+from models_library.api_schemas_catalog import ServiceAccessRightsGet
+from models_library.services import ServiceKey, ServiceVersion
+from models_library.services_db import ServiceAccessRightsAtDB
+
+from ...db.repositories.services import ServicesRepository
+from ...models.schemas.constants import RESPONSE_MODEL_POLICY
+from ..dependencies.database import get_repository
+from ..dependencies.services import AccessInfo, check_service_read_access
+
+#
+# Routes -----------------------------------------------------------------------------------------------
+#
+
+router = APIRouter()
+logger = logging.getLogger(__name__)
+
+
+@router.get(
+    "/{service_key:path}/{service_version}/accessRights",
```

Comment:
I think you can go with access_rights here... these are internal APIs only used by python

Reply:
I guess either I would leave it like this, based on https://google.aip.dev/122 it should be camelCase, or on the other hand to be consistent with our other historical api's it should be `access-rights`. But in the catalog service, we do not have this situation yet, so we can go with the google aip?

Reply:
well I think I do agree with you.